packages/driver/src/cypress/log.ts

```diff
@@ -374,16 +374,16 @@ export class Log {
     }
 
     if (this.config('experimentalSessionAndOrigin') && !Cypress.isCrossOriginSpecBridge) {
-      const activeSpecBridgeOriginPolicyIfApplicable = this.state('currentActiveOriginPolicy') || undefined
+      const activeSpecBridgeOriginIfApplicable = this.state('currentActiveOrigin') || undefined
       // @ts-ignore
-      const { originPolicy: originPolicyThatIsSoonToBeOrIsActive } = Cypress.Location.create(this.state('anticipatingCrossOriginResponse')?.href || this.state('url'))
+      const { superDomainOrigin: originThatIsSoonToBeOrIsActive } = Cypress.Location.create(this.state('anticipatingCrossOriginResponse')?.href || this.state('url'))
 
-      if (activeSpecBridgeOriginPolicyIfApplicable && activeSpecBridgeOriginPolicyIfApplicable === originPolicyThatIsSoonToBeOrIsActive) {
+      if (activeSpecBridgeOriginIfApplicable && activeSpecBridgeOriginIfApplicable === originThatIsSoonToBeOrIsActive) {
         Cypress.emit('request:snapshot:from:spec:bridge', {
           log: this,
           name,
           options,
-          specBridge: activeSpecBridgeOriginPolicyIfApplicable,
+          specBridge: activeSpecBridgeOriginIfApplicable,
```

Comment:
This we definitely need, or at least a way to know the origin of the currently active spec bridge.

Reply:
I wonder if we want to make a issue for next sprint to audit the multi origin code and do some clean up to keep up with all the changes we've made

Reply:
I wonder if this even works in it's current state

Reply:
I am thinking we might be able to use `currentActiveSuperDomainOrigin` and get the `superDomainOrigin` of the `url` in `state` to see where we can send the spec bridge. Probs out of scope for this PR, but it definitely seems like a way we can fix that behavior. I think it's currently broken (which isn't a huge deal IMO)

Reply:
actually I can't read. This was a simple fix and just removed it in e66aa928e9